### DISCUSSION TITLE
希望能有一个视频合集倒序或者按更新日期排序的功能

### DISCUSSION THREAD

**Discussion by 233YUZI** [the1812/Bilibili-Evolved#4212] [功能建议 / Ideas]
很多up主的视频合集喜欢把最新一期视频放在上面，导致自动连播的时候会播放到上一集，就很反人类
![image](https://github.com/the1812/Bilibili-Evolved/assets/69708373/c6575dab-aa3c-4db9-a0a5-322ba2857184)

**Comment by Geeyun-JY3** [comment]
请善用搜索，重复：#2368、https://github.com/the1812/Bilibili-Evolved/discussions/1930#discussioncomment-849494
